api-client/src/models/LocationModel.ts

```diff
@@ -1,51 +1,40 @@
-import { Position } from 'geojson'
-
-class LocationModel {
+class LocationModel<T> {
```

Comment:
do we expect different types than number for long/lat or why do we need a generic here. Maybe i overseen sth

Reply:
long/lat may be null for the locations of events. therefore the generic should only ever either be `number` or `number | null`. To avoid type checks for pois I introduced this generic such that it is guaranteed that pois always have long/lat defined while it is not guaranteed for events.